src/functions/wiki-documentation/find-article/index.ts

```diff
@@ -0,0 +1,16 @@
+import { handlerPath } from '@libs/handler-resolver';
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      httpApi: {
+        method: 'get',
+        path: '/articles/{path+}',
```

Comment:
Find endpoints should not accept path. Instead they should use ID, so instead of doing this you should have 

list endpoint with query parameter path:

    /articles?path=something

and find endpoint with:

    /articles/{articleId}